### PR TITLE
8 - ODS component - cards - image first card

### PR DESCRIPTION
**Pull request by paulinea**

### CODE REVIEW

**Review by ludovic35 — CHANGES_REQUESTED** [review]

File: app/lib/ui/components/cards/card_vertical_image_first.dart [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import 'package:flutter/material.dart';
```
[the remark this answers is not in the document]
Comment (paulinea) [reply app/lib/ui/components/cards/card_vertical_image_first.dart:1]: Not sure about this one, I think we chose to remove the `component` prefix to have lighter names. But we have to do this progressively.

File: library/lib/components/card/ods_vertical_image_first_card.dart [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  final String title;
+
+  /// The optional card's subtitle displayed below the title.
+  final String? subtitle;
+
+  /// The optional card's text.
+  final String? text;
+
+  /// Optional text of the first button in the card. If not present, button will not be shown. If present, [onButton1Click] need to be handled.
+  final String? button1Text;
```
[the remark this answers is not in the document]
Comment (paulinea) [reply library/lib/components/card/ods_vertical_image_first_card.dart:40]: ok, it's a good idea

File: app/lib/ui/components/ods_components.dart [hunk 3]
```diff
@@ -1,25 +1,15 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_gen/gen_l10n/ods_flutter_app_localizations.dart';
+import 'package:ods_flutter_demo/ui/components/cards/card_vertical_image_first.dart';
 import 'package:ods_flutter_demo/ui/components/cards/component_card_small.dart';
 import 'package:ods_flutter_demo/ui/components/checkboxes/component_checkboxes.dart';
-import 'package:ods_flutter_demo/ui/components/component.dart';
+import 'package:ods_flutter_demo/ui/components/component_entities.dart';
```
[the remark this answers is not in the document]
Comment (paulinea) [reply app/lib/ui/components/ods_components.dart:6]: Actually, the list of components could be in `components_screen`. What do you think?

[the remark this answers is not in the document]
Comment (ludovic35) [reply app/lib/ui/components/ods_components.dart]: Good idea

**Review by ludovic35 — CHANGES_REQUESTED** [review]

File: library/lib/components/utilities/ods_button_entity.dart [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  const OdsButtonEntity({
+    required this.label,
+    required this.onClick,
+  });
+
+  /// The text label for this button.
+  final String label;
+
+  /// The action executed on button click.
+  final VoidCallback? onClick;
```
[the remark this answers is not in the document]
Comment (paulinea) [reply library/lib/components/utilities/ods_button_entity.dart:15]: a mistake, I fix it

**Review by ludovic35 — APPROVED** [review]

**Pull request merged**